pkg/virt-handler/vm.go

```diff
@@ -383,6 +394,28 @@ func domainMigrated(domain *api.Domain) bool {
 	return false
 }
 
+func getPodInterfacefromFileCache(vmi *v1.VirtualMachineInstance, ifaceName string) (*network.PodCacheInterface, error) {
```

Comment:
You'll want to add this as a non-exported function for the VirtualMachineController object so you have access the the map and lock.

something like...

```
func (d *VirtualMachineController) getPodInterfacefromFileCache(vmi *v1.VirtualMachineInstance, ifaceName string) (*network.PodCacheInterface, error) {
```

Reply:
Corrected now